src/js/components/modals/AlertModalComponent.jsx

```diff
@@ -12,7 +12,8 @@ var AlertModalComponent = React.createClass({
   },
 
   componentDidMount: function () {
-    React.findDOMNode(this.refs.button).focus();
+    let refs = this.refs;
+    React.findDOMNode(refs.button).focus();
```

Comment:
I reckon `refs` isn't needed anymore, since we kicked out the other `findDOMNode`s – want me to take care of this @philipnrmn ?